pom.xml

```diff
@@ -63,6 +63,8 @@
     <commons.osgi.private />
     <commons.jira.id>BCEL</commons.jira.id>
     <commons.jira.pid>12314220</commons.jira.pid>
+    <mockito.version>5.8.0</mockito.version>

```

Comment:
Hello @gtoison 
This version of Mockito depends on Java 11 and this project's baseline is Java 8. Please use a Java 8-based version. We can discuss separately raising the platform's Java requirement from Java 8 to 11. If you want to start this discussion, please do so on the developer's mailing list. see https://commons.apache.org/mail-lists.html

Reply:
I've just pushed a commit to use Mockito 4 (so it should be compatible with java 8) and made the adjustments to the test because final methods/classes can't be mocked with Mockito 4

Reply:
OK, green build.